user_sync/rules.py

```diff
@@ -780,14 +812,14 @@ def read_remove_list(file_path, delimiter = None, logger = None):
             id_type = row.get(id_type_column_name)
             user = row.get(user_column_name)
             domain = row.get(domain_column_name)
-            user_key = RuleProcessor.get_user_key(user, domain, id_type)
+            user_key = RuleProcessor.get_user_key(user, domain, None, id_type)
```

Comment:
I added the None parameter because get_user_key requires 4 parameters, and read_remove_list was crashing when it got to this point... the email parameter for get_user_key doesn't seem to be required, so I set the argument to None for now...